src/chat/commands/so.ts

```diff
@@ -0,0 +1,36 @@
+import { OnCommandEvent, OnSayEvent } from "../../models"
+import { EventBus, Events } from "../../events"
+import { ShouldThrottle } from '../shouldThrottle'
+
+/**
+ * Sends a message to chat with info on BBB Instagram
+ * @param onCommandEvent 
+ */
+export function So(onCommandEvent: OnCommandEvent): void {
+
+  const cooldownSeconds = 300
+
+  // The broadcaster is allowed to bypass throttling. Otherwise,
+  // only proceed if the command hasn't been used within the cooldown.
+  if (!onCommandEvent.flags.broadcaster &&
```

Comment:
Let's remove the throttling on this command. Since it's mod/broadcaster only, I think we're safe to have it called more often.